misc/python/materialize/mzbuild.py

```diff
@@ -237,14 +238,26 @@ def build(self) -> None:
                 for d in self.extract.get(package, []):
                     shutil.copy(Path(message["out_dir"]) / d, self.path / Path(d).name)
 
+        if self.assets_dir:
+            shutil.rmtree(self.path / "assets_dir", ignore_errors=True)
+            shutil.copytree(self.rd.root / self.assets_dir, self.path / "assets_dir")
+
     def run(self) -> None:
         super().run()
         self.build()
 
     def inputs(self) -> Set[str]:
         crate = self.rd.cargo_workspace.crate_for_bin(self.bin)
         deps = self.rd.cargo_workspace.transitive_path_dependencies(crate)
-        return super().inputs() | set(inp for dep in deps for inp in dep.inputs())
+        inputs = super().inputs() | set(inp for dep in deps for inp in dep.inputs())
+
+        if self.assets_dir:
+            inputs = inputs | set(
+                self.rd.root / self.assets_dir / f
+                for f in os.listdir(self.rd.root / self.assets_dir)
+            )
+
+        return inputs
```

Comment:
inputs will now contain both relative paths from cargo and absolute paths from assets_dir